HOWTO-ar.md

```diff
@@ -1,4 +1,4 @@
-إقرأ هذا بلغات أخرى: [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [Deutsch](HOWTO-de.md), [Bahasa Indonesia](HOWTO-id.md), [Українська](HOWTO-uk_UA.md).
+إقرأ هذا بلغات أخرى: [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [Deutsch](HOWTO-de.md), [Bahasa Indonesia](HOWTO-id.md), [Українська](HOWTO-uk.md).
```

Comment:
With the intention of avoiding merge conflicts in other pending PRs, I'm in favour of avoid this kind of crosslinks in all changed files. **Try to scope to ​​`-uk`** files.

Reply:
@davorpa In any case you will need to add those references in other files. Wouldn't it be better to do it right now?